resources/assets/coffee/react/beatmapset-page/difficulty-chart.coffee

```diff
@@ -29,11 +29,11 @@ class BeatmapsetPage.DifficultyChart extends React.Component
 
   _renderChart: ->
     data = [
-      {value: @props.beatmap.cs, label: osu.trans 'beatmaps.beatmapset.show.stats.chart.cs'},
-      {value: @props.beatmap.drain, label: osu.trans 'beatmaps.beatmapset.show.stats.chart.hp'}
-      {value: @props.beatmap.accuracy, label: osu.trans 'beatmaps.beatmapset.show.stats.chart.od'},
-      {value: @props.beatmap.ar, label: osu.trans 'beatmaps.beatmapset.show.stats.chart.ar'},
-      {value: @props.beatmap.difficulty_rating, label: osu.trans 'beatmaps.beatmapset.show.stats.chart.sd'},
+      {value: @props.currentBeatmap.cs, label: osu.trans 'beatmaps.beatmapset.show.stats.chart.cs'},
```

Comment:
This component doesn't seem to need differentiating whether the beatmap is `currentBeatmap` or just any `beatmap`?